imap_processing/mag/l1b/mag_l1b.py

```diff
@@ -0,0 +1,112 @@
+"""MAG L1B Processing."""
+
+from pathlib import Path
+
+import numpy as np
+import xarray as xr
+
+from imap_processing.cdf.imap_cdf_manager import ImapCdfAttributes
+from imap_processing.cdf.utils import load_cdf
+
+
+def mag_l1b(input_dataset: xr.Dataset, version: str) -> xr.Dataset:
```

Comment:
I like that you're passing Datasets in/out, seems like a nice clean way to do it!